cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -962,19 +1014,25 @@ def loadbalancer_kubeconfig(loadbalancer, ca, client):
 @when('certificates.ca.available', 'certificates.client.cert.available',
       'authentication.setup')
 @when_not('loadbalancer.available')
-def create_self_config(ca, client):
+def create_self_config():
```

Comment:
This is called here, which needs to be updated: https://github.com/juju-solutions/kubernetes/blob/a15b713653127a5fa515ccfa4bbc03561eccea2d/cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py#L2173